e_chil.c

```diff
@@ -224,8 +166,10 @@ static RAND_METHOD hwcrhk_rand = {
 /* Constants used when creating the ENGINE */
 static const char *engine_hwcrhk_id = "chil";
 static const char *engine_hwcrhk_name = "CHIL hardware engine support";
+#  ifndef OPENSSL_NO_DYNAMIC_ENGINE
 /* Compatibility hack, the dynamic library uses this form in the path */
 static const char *engine_hwcrhk_id_alt = "ncipher";
+#  endif
```

Comment:
There's no reason to add this guard.  This gets built as a module, no exception.

Reply:
Removed